proto/lavanet/lava/plans/plan.proto

```diff
@@ -23,6 +23,7 @@ message Plan {
     string type = 12 [(gogoproto.jsontag) = "type"]; // plan type
     uint64 annual_discount_percentage = 13 [(gogoproto.jsontag) = "annual_discount_percentage"]; // discount for buying the plan for a year
     Policy plan_policy = 14 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "plan_policy"];
+    uint64 projects = 15 [(gogoproto.jsontag) = "projects"]; // number of allowed projects
```

Comment:
can you rename this?
maybe projects_limit?

Reply:
done. see https://github.com/lavanet/lava/commit/d8cbc30eb943844a5306b0300c9dbd7bcc45bb45